src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -207,26 +208,179 @@
+                                                <Run x:Uid="General_SettingsBackupAndRestoreDescription" />
+                                    </TextBlock>
+                                </controls:Setting.Description>
+                                <controls:Setting.ActionContent>
+                                    <Grid Visibility="Visible">
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="1*" />
+                                            <ColumnDefinition Width="1*" />
+                                        </Grid.ColumnDefinitions>
+                                        <Button Grid.Column="0" x:Uid="General_SettingsBackupAndRestore_ButtonBackup"
+                                        Command="{Binding BackupConfigsEventHandler}"
+                                        IsEnabled="True"
+                                        Visibility="Visible"
+                                        HorizontalAlignment="Right"/>
+
+                                        <Button Grid.Column="2" x:Uid="General_SettingsBackupAndRestore_ButtonRestore"
+                                        Command="{Binding RestoreConfigsEventHandler}"
+                                        IsEnabled="True"
+                                        Visibility="Visible"
+                                        HorizontalAlignment="Right"/>
+
+                                    </Grid>
+                                </controls:Setting.ActionContent>
+                            </controls:Setting>
+                        </controls:SettingExpander.Header>
+
+                        <controls:SettingExpander.Content>
+
+
+                            <StackPanel Orientation="Vertical">
+
+                                <controls:Setting x:Uid="General_SettingsBackupAndRestoreLocationText" Visibility="Visible" Style="{StaticResource ExpanderContentSettingStyle}">
+
+                                    <controls:Setting.ActionContent>
+
+                                        <Grid Padding="10,5,0,0">
+                                            <Grid.ColumnDefinitions>
+                                                <ColumnDefinition Width="Auto" />
+                                                <ColumnDefinition Width="*" />
+                                                <ColumnDefinition Width="Auto" />
+                                            </Grid.ColumnDefinitions>
+                                            <Grid.RowDefinitions>
+                                                <RowDefinition Height="Auto" />
+                                            </Grid.RowDefinitions>
+                                            <TextBlock MaxWidth="350" Grid.Row="0" Grid.Column="1" Text="{x:Bind Mode=TwoWay, Path=ViewModel.SettingsBackupAndRestoreDir}" IsTextSelectionEnabled="True" MinWidth="350" />
```

Comment:
@jefflord 
Let's use secondary text style and we should enable text wrapping because we have a maximal width.

@niels9001, @jefflord 
Can we set the tect to be vertical centered?

Reply:
> @niels9001, @jefflord 
Can we set the text to be vertical centered?

@jefflord
Let's try `VerticalAlignment="Center"`

Reply:
I made some changes and I think @niels9001 wants to do a final round updates later.